ui.tests/test-module/specs/panelcontainer/panelcontainer.runtime.complex.spec.js

```diff
@@ -292,6 +292,16 @@ describe( "Form Runtime with Panel Container complex repeatability use cases ",
         return innerPromise;
     };
 
+    const checkAddRemoveButton = (instanceManager, count, buttonId) => {
+        let resolution = undefined;
+        const promise = new Cypress.Promise((resolve, reject) => {resolution = resolve;});
+        cy.get(`#${buttonId} button`).click().then(() => {
+            const e = checkInstanceHTML(instanceManager, count);
+            resolution(e);
+        });
+        return promise;
```

Comment:
i can't find this promise's resolve method, how is the resolution call getting handled.

Reply:
resolution

Reply:
Resoultion is set as undefined,

Reply:
line number 300, updates it.

Reply:
You may find the resolution similarly used at  https://github.com/adobe/aem-core-forms-components/blob/dev/ui.tests/test-module/specs/panelcontainer/panelcontainer.runtime.complex.spec.js#L275